### PR TITLE
Remove web/proxy containers

### PR DESCRIPTION
**Pull request by bo0tzz**
To be merged when release 1.88.0 goes out

### CODE REVIEW

File: charts/immich/values.yaml [hunk 1]
```diff
@@ -14,12 +14,9 @@ env:
   TYPESENSE_ENABLED: "{{ .Values.typesense.enabled }}"
   TYPESENSE_API_KEY: "{{ .Values.typesense.env.TYPESENSE_API_KEY }}"
   TYPESENSE_HOST: '{{ printf "%s-typesense" .Release.Name }}'
-  IMMICH_WEB_URL: '{{ printf "http://%s-web:3000" .Release.Name }}'
-  IMMICH_SERVER_URL: '{{ printf "http://%s-server:3001" .Release.Name }}'
-  IMMICH_MACHINE_LEARNING_URL: '{{ printf "http://%s-machine-learning:3003" .Release.Name }}'
```
Comment (jrasm91) [inline charts/immich/values.yaml:19]: This one is still in use I believe, even though you can set it from the gui, but I am also fine removing it

**Review by jrasm91 — APPROVED** [review]
Lgtm

File: charts/immich/values.yaml (hunk 1 above)
Comment (bo0tzz) [reply charts/immich/values.yaml:19]: Good catch, thanks! It's no longer documented, should we put it back into the docs pages then?

Comment (jrasm91) [reply charts/immich/values.yaml:19]: It is a weird one. It is part of the config and so I just want to remove it, but it is still here for backwards compatibility with the AIO. I think it is fine to leave undocumented for now.

**Review by uhthomas — APPROVED** [review]
Looks okay to me - as long as it's been tested, I think this is fine to merge.

**Comment by bo0tzz** [comment]
I do still need to do some testing, let me switch this to draft for now. (If anybody wants to help with testing, that would be welcome!)

**Comment by Emilien-M** [comment]
> I do still need to do some testing, let me switch this to draft for now. (If anybody wants to help with testing, that would be welcome!)

I just updated to v1.88.0 and it's work like a charm.

**Pull request merged**